packages/fluent/scss/pdf-viewer/_layout.scss

```diff
@@ -0,0 +1,177 @@
+@use "../core/color-system" as *;
+@use "_variables.scss" as *;
+
+@mixin kendo-pdf-viewer--layout() {
+
+    .k-pdf-viewer {
+        border-width: var( --kendo-pdf-viewer-border-width, #{$kendo-pdf-viewer-border-width} );
+        border-style: solid;
+        box-sizing: border-box;
+        position: relative;
+        outline: 0;
+        font-family: var( --kendo-pdf-viewer-font-family, #{$kendo-pdf-viewer-font-family} );
+        font-size: var( --kendo-pdf-viewer-font-size, #{$kendo-pdf-viewer-font-size} );
+        line-height: var( --kendo-pdf-viewer-line-height, #{$kendo-pdf-viewer-line-height} );
+        display: flex;
+        flex-direction: column;
+        overflow: hidden;
+        -webkit-touch-callout: none;
+        -webkit-tap-highlight-color: $rgba-transparent;
```

Comment:
use css var here

Reply:
this is a constant, thus there is no need of utilizing a css var here